core/modules/qmeta/QMetaMysql.cc

```diff
@@ -0,0 +1,569 @@
+/*
+ * LSST Data Management System
+ * Copyright 2015 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "qmeta/QMetaMysql.h"
+
+// System headers
+#include <algorithm>
+
+// Third-party headers
+#include "boost/lexical_cast.hpp"
+
+// LSST headers
+#include "lsst/log/Log.h"
+
+// Qserv headers
+#include "Exceptions.h"
+#include "QMetaTransaction.h"
+#include "sql/SqlResults.h"
+
+namespace {
+
+// logger instance for this module
+LOG_LOGGER _logger = LOG_GET("lsst.qserv.qmeta.QMetaMysql");
+
+}
+
+namespace lsst {
+namespace qserv {
+namespace qmeta {
+
+// Constructors
+QMetaMysql::QMetaMysql(mysql::MySqlConfig const& mysqlConf)
+  : QMeta(),
+    _conn(mysqlConf)
+{
```

Comment:
I'm not fond of them either...  Maybe we can revisit/revise them since we have a lot of new people on the team these days.  Some departure from the LSST line is probably (?) okay, as long as its applied consistently throughout the qserv codebase?